### ISSUE TITLE
Failing during initialisation. undefined offset

### ISSUE THREAD

**Issue by bendubuisson** [FoxxMD/laravel-elasticbeanstalk-cron#5]
So I get a `[ErrorException]     
  Undefined offset: 0`

when deploying to EB.

Looks like [https://github.com/FoxxMD/laravel-elasticbeanstalk-cron/blob/master/src/Console/AWS/ConfigureLeaderCommand.php#L53-L58](https://github.com/FoxxMD/laravel-elasticbeanstalk-cron/blob/master/src/Console/AWS/ConfigureLeaderCommand.php#L53-L58)

```
 $tags = $info = $this->ecClient->describeInstances(['Filters' => [
                    ['Name'   => 'instance-id',
                     'Values' => [$result]
                    ]
                ]
                                                                   ])->get('Reservations')[0]['Instances'][0]['Tags'];
```

is the issue. 

Any thoughts?

**Comment by bendubuisson** [comment]
I have dumped the result of `get->('Reservations)` and it is an empty array. Am I missing something?

**Comment by bendubuisson** [comment]
right, got it. You have the region hardcoded.

**Comment by bendubuisson** [comment]
And I see a PR that would fix it: #3  Any chance you could merge it?

Cheers

**Comment by FoxxMD** [comment]
Merged in d95a768a5e1a3399f750b67209b41e3d21425c4a
